dev/breeze/pyproject.toml

```diff
@@ -45,29 +45,28 @@ classifiers = [
 requires-python = "~=3.8"
 
 dependencies = [
+    "PyGithub>=2.1.1",
```

Comment:
Does not really matter, but since PyPI names are case insensitive, I’d put this back down and convert it to lower case like the rest of the entries.

Reply:
Yep. Good idea.  Normalizing is a good idea. I got already warnings from `flit` about normalizing the `.rc` / `.dev` - we did not have the `.` - so I fixed that in this change. I will fix the casing here too (and it will make it nicer for sorting too... It disturbed me a bit that it was first on the list :)

Reply:
Fixed.